source/Octostache/Templates/BuiltInFunctions.cs

```diff
@@ -25,7 +25,8 @@ static class BuiltInFunctions
             {"replace", TextReplaceFunction.Replace },
             {"substring", TextSubstringFunction.Substring},
             {"truncate", TextManipulationFunction.Truncate},
-            {"trim", TextManipulationFunction.Trim}
+            {"trim", TextManipulationFunction.Trim},
+            {"uripart", TextManipulationFunction.UriPart}
```

Comment:
I've seen a few terms used for components of a URI. "Components" and "Parts" are the most common. I like "uripart" because it's terse and clear.

Reply:
It's more human friendly, and as you say terse.